editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1519,8 +1519,21 @@ void CanvasItemEditor::_viewport_gui_input(const Ref<InputEvent> &p_event) {
 			Vector2 minsize = canvas_item->edit_get_minimum_size();
 
 			if (uniform) {
+				// Keep the height/width ratio of the item
 				float aspect = local_rect.size.aspect();
 				switch (drag) {
+					case DRAG_LEFT: {
+						drag_vector.y = -drag_vector.x / aspect;
+					} break;
```

Comment:
wat 😕 
`} break;`
Why is this break outside of `}`, and even worse - in the same line as `}`?
@akien-mga is this official style for Godot?